.github/workflows/sync-board.yaml

```diff
@@ -3,6 +3,8 @@ name: sync-board
 # FYI secrets.ADD_TO_PROJECT_PAT is a personal access token belonging to @akeller.
 
 on:
+  pull_request:
+    types: [review_requested]
   pull_request_target:
     types: [review_requested]
```

Comment:
❓ Are you sure about this?

I believe the previous commit did not trigger the workflow, because
- `pull_request` does not allow the use of secrets in pull requests originating from forks
- `pull_request_target` runs in the base (i.e. target) of the pull request, not the head. This means that the workflow must exist with `pull_request_target` on `main` in order for it to trigger from a forked pull request.

🔧 IMO, `on: pull_request` is not needed, but the change to `pull_request_target` will only take effect after this pull request has been merged.

Reply:
> ❓ Are you sure about this?

I'm definitely not sure! I was surprised to see it not run on `pull_request_target` for an internal PR but your explanation totally totally makes sense! 

I'll fix this before merging.

Reply:
Reverted in the latest commit!